tests/functional-utils/test_type_inference.py

```diff
@@ -18,51 +22,52 @@ def check_mypy_run(
     expected_returncode: int = 1,
 ) -> None:
     """Helper to run mypy and check the output."""
-    out, err, returncode = mypy.api.run(cmd_line)
+    out, err, returncode = api.run(cmd_line)
     assert out == expected_out, err
     assert err == expected_err, out
     assert returncode == expected_returncode, returncode
 
 
 # The following tests all run code snippets through mypy that contain a
 # `reveal_type` statement, and then match the mypy output against our expectations
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="requires Python 3.8 or higher")
```

Comment:
The tests fail with py37, but run fine with everything else. Set this skipif as 37 is going away relatively soon.